kernelci/cli/user.py

```diff
@@ -49,7 +50,7 @@ def find(attributes, config, api, indent, secrets):
     api = get_api(config, api, secrets)
     attributes = split_attributes(attributes)
     users = api.get_users(attributes)
-    data = json.dumps(users, indent=indent)
+    data = json.dumps(users, indent=indent or None)
```

Comment:
same comment as above.

Reply:
Of course, I fixed this everywhere :)  And then wrapped it up in a helper function.